src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -59,6 +59,8 @@
     public final IPlayerContext ctx;
 
     public ExampleBaritoneControl(IBaritone baritone) {
+        if (thisExampleBaritoneControl == null)
```

Comment:
no, we d oNot do this, each baritone instance is 100% segmented from the other. no static variables **anywhere**. make a field in Baritone

Reply:
"no, we do not do this,"

Reply:
I'm not sure about how you want me to do this.

Reply:
Sorry, I changed my mind about this, see the last comment. I also said `read the last comment first since i changed my mind like half way through lol`. Make this class listen for a ProcessCompletionEvent